src/services/featurehelper.js

```diff
@@ -436,15 +436,33 @@ ngeo.FeatureHelper.prototype.getVertexStyle = function(opt_incGeomFunc) {
         return;
       }
 
-      var coordinates;
+      var innerMultiCoordinates;
+      var multiCoordinates = [];
+      var coordinates = [];
+      var i, ii;
       if (geom instanceof ol.geom.LineString) {
-        coordinates = feature.getGeometry().getCoordinates();
-        return new ol.geom.MultiPoint(coordinates);
+        coordinates = geom.getCoordinates();
+      } else if (geom instanceof ol.geom.MultiLineString) {
+        multiCoordinates = geom.getCoordinates();
       } else if (geom instanceof ol.geom.Polygon) {
-        coordinates = feature.getGeometry().getCoordinates()[0];
+        coordinates = geom.getCoordinates()[0];
+      } else if (geom instanceof ol.geom.MultiPolygon) {
+        innerMultiCoordinates = geom.getCoordinates();
+      }
+
+      if (innerMultiCoordinates) {
+        for (i = 0, ii = innerMultiCoordinates.length; i < ii; i++) {
```

Comment:
To my knowledge, plain `for` loops are faster than `forEach` because in the latter it requires anonymous functions and often access to variables that are outside its scope (which has a heavy cost in JavaScript).  That anonymous function is also created on each iteration.

`ii` isn't really required indeed, but is a faster way to do this that `for (i = 0; i < innerMultiCoordinates.length; i++)` because the lenght is calculated on each iteration.  Putting it in a variable first makes the execution of the code faster.

Is it okay if I leave this like this?